packages/ui/AnnounceDecorator/useAnnounce.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import Announce from './Announce';
+
+/**
+ * Object returned by `useAnnounce`
+ *
+ * @typedef {Object} useAnnounceInterface
+ * @property {Function} announce Called to alert the user of behavior for accessibility
+ * @property {Node}     children An additional element which must be added to the component tree to
+ *                               support notifying the user
+ * @private
+ */
+
+/**
+ * Provides a method to alert the user of behavior for accessibility.
+ *
+ * ```
+ * 	function Component () {
+ *		const {announce, children} = useAnnounce();
+ *
+ *		return (
+ *			<Base onClick={() => announce('Unusually important notification for accessibility')}>
+ *				{children}
+ *			</Base>
+ *		);
+ *	}
+ * ```
+ *
+ * @returns {useAnnounceInterface}
+ * @private
+ */
+function useAnnouce () {
+	const ref = React.useRef(null);
+	const announce = React.useCallback(message => {
+		if (ref && ref.current) {
```

Comment:
Could we remove the `ref &` because the useRef returns {current: null} and the Boolean(`ref`) is always true ?

Reply:
Yep. You are correct!

Reply:
Seeing if I can "suggest" to myself to update it ... :)

```suggestion
		if (ref.current) {
```